darwin/dataset/utils.py

```diff
@@ -238,102 +285,97 @@ def get_coco_format_record(
     image_id: Optional[Union[str, int]] = None,
     classes: Optional[List[str]] = None,
 ) -> Dict[str, Any]:
-    """
```

Comment:
What was the reason behind removing the docblock?  Internal function?

Reply:
That's strange, its there in my code. Will see if I can get it back on the github as well.